tuf/ngclient/updater.py

```diff
@@ -200,6 +204,8 @@ def download_target(
                 ) from e
 
             filepath = os.path.join(destination_directory, target_filepath)
+            if self.config.consistent_snapshot:
+                filepath = os.path.join(destination_directory, orig_target_path)
```

Comment:
Storing `orig_target_path` is unnecessary, you can retrieve it from  `targetinfo["filepath"]` instead.
This way the  second `if self.config.consistent_snapshot` condition becomes redundant too.

Reply:
Agree, the only downside is that it will require more lines of code than just storing it in `orig_target_path`, but it will make it
easier to understand - one variable less.